frontend/public/locales/en/booking.json

```diff
@@ -1,8 +1,8 @@
 {
-    "startDate": "Start date",
-    "startTime": "Start time",
-    "endDate": "End date",
-    "endTime": "End time",
+    "startDate": "Start Date",
+    "startTime": "Start Time",
+    "endDate": "End Date",
+    "endTime": "End Time",
```

Comment:
Why is the second word capitalized?

Reply:
Hmm I was thinking as a header in english everything should be capitalized, but I googled on "start date" in google images and saw lots of people writing "Start date", so I'll changei t.